lib/config/locales/en.yml

```diff
@@ -174,12 +176,21 @@ en:
             Refer to:
             https://docs.mongodb.com/mongoid/current/reference/fields/#custom-field-options"
         invalid_field_type:
-          message: "Invalid field type %{type_inspection} for field '%{field}' on model '%{klass}'."
-          summary: "Model '%{klass}' defines a field '%{field}' with an unknown type value
-            %{type_inspection}."
-          resolution: "Please provide a valid type value for the field.
+          message: "Invalid field type %{type_inspection} for field :%{field} on model '%{klass}'."
```

Comment:
Please review the language changes as well and undo the quote removal here.

Reply:
Resolved.